.travis.yml

```diff
@@ -7,6 +7,9 @@ env:
 - RUNJOB=jshint
 - RUNJOB=qunit
 
+before_script:
+  - nvm install 4
+
 script: bash ./build/travis/script.sh
```

Comment:
I think this should be replaced with `composer test`, and the .sh files deleted, just as in all other code repositories we maintain. But this can be done in a later patch.